scripts/aws/create-kubernetes-cluster.go

```diff
@@ -226,6 +227,11 @@ func createEksWorkerNodes(cfClient *cloudformation.CloudFormation, nodesStackNam
 
 	s := string(sf)
 
+	// Base image for Amazon EKS worker nodes
+	// with Kubernetes version 1.12.7
+	// for region us-east-2
+	eksAmi := aws.String("ami-04ea7cb66af82ae4a")
```

Comment:
Should we move this to some kind of config?

Reply:
Should we comment at least there to get it?

https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html